diff --git a/node_modules/react-native-razorpay/android/build.gradle b/node_modules/react-native-razorpay/android/build.gradle
index 050a6d8..98b6930 100644
--- a/node_modules/react-native-razorpay/android/build.gradle
+++ b/node_modules/react-native-razorpay/android/build.gradle
@@ -19,11 +19,12 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    namespace "com.razorpay.rn"
+    compileSdkVersion safeExtGet('compileSdkVersion', 35)
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 28)
+        minSdkVersion safeExtGet('minSdkVersion', 26)
+        targetSdkVersion safeExtGet('targetSdkVersion', 35)
         versionCode 1
         versionName "1.0"
 
diff --git a/node_modules/react-native-razorpay/react-native-razorpay.podspec b/node_modules/react-native-razorpay/react-native-razorpay.podspec
index 113ef82..8f5fe33 100644
--- a/node_modules/react-native-razorpay/react-native-razorpay.podspec
+++ b/node_modules/react-native-razorpay/react-native-razorpay.podspec
@@ -11,8 +11,7 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platform     = :ios, "10.0"
-  s.ios.deployment_target = '10.0'
+  s.platforms = { :ios => "15.1" }
 
   s.source       = { :git => "https://github.com/razorpay/react-native-razorpay.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m,mm}"
